mongodb/src/main/scala/akka/stream/alpakka/mongodb/MongoConnectionDetails.scala

```diff
@@ -0,0 +1,9 @@
+package akka.stream.alpakka.mongodb
```

Comment:
All files need the copyright header, see `akka.stream.alpakka.file.scaladsl.DirectoryChangesSource` for example.